contents/service-message.md

```diff
@@ -5,49 +5,52 @@ showSidebar: true
 noindex: true
 ---
 
-import { StaticImage } from 'gatsby-plugin-image'
-import Table from './_includes/example-table.mdx'
-import { getImage, GatsbyImage } from 'gatsby-plugin-image'
 
 ![Builder Hog](./images/service-message/worker-hog.png)
```

Comment:
I tried using `img` for this so that we can set the dimensions of the image (it's is currently too large and so a bit blurry on retina displays) – but Gatsby didn't want to set the right `src` value. Then tried `StaticImage`, but that just did not want to render at all. What's going on?

Reply:
Unfortunately, you can't use `StaticImage` within Markdown/MDX, but there is a workaround!

```
---
title: Scheduled Maintenance
showTitle: false
showSidebar: true
noindex: true
images:
    - './images/service-message/worker-hog.png'
---

import { getImage, GatsbyImage } from 'gatsby-plugin-image'


<GatsbyImage image={getImage(props.images[0])} alt="Worker hog" />
```